docs/injector-example.js

```
@@ -0,0 +1,351 @@
+                this.logger = (__runInitializers(this, _api_extraInitializers), __runInitializers(this, _logger_initializers, void 0));
+                __runInitializers(this, _logger_extraInitializers);
+            }
+            DecoratedService.prototype.fetchData = function (id) {
+                this.logger.info('DecoratedService', "Fetching data for ".concat(id));
+                this.api.request('netease', '/song/detail', { ids: id });
+            };
+            return DecoratedService;
+        }()),
+        (function () {
+            var _metadata = typeof Symbol === "function" && Symbol.metadata ? Object.create(null) : void 0;
+            _api_decorators = [(0, decorators_1.inject)(IApiService)];
```

ESLint

[error] 'IApiService' is not defined.

(no-undef)